packages/sanes-chrome-extension/src/logic/persona/persona.ts

```diff
@@ -96,7 +119,38 @@ export class Persona {
     return out;
   }
 
+  public startSigningServer(): UseOnlyJsRpcSigningServer {
```

Comment:
Move this, into a separate class, something like personaSignaling or something like that. Persona has enough keeping the props for interacting with chains and exposing methods for feeding react views. Don't you think?

Reply:
Since the `Persona` instance owns the `MultiChainSigner` instance without exposing it, the signing server also needs to live here. The server is just a wrapper around `MultiChainSigner` that allows processing commands from outside.